_episodes/09-pull-requests.md

```diff
@@ -55,25 +85,38 @@ Unpacking objects: 100% (6/6), done.
 ~~~
 {: .output}
 
-Next we will set up a connection or remote to the authoritative repository.
-In your browser, you can get back to this repo by clicking on the link in the upper left hand corner,
-under your username and repo.
+Next we will set up a connection or **remote** to the authoritative repository 
+(the original version given to you by your instructor).
+In your browser, you can go back this repo by clicking on the link that says 'forked from'
+in the upper left hand corner, under your username and repo name.
 
-Copy the web address for this repo (from the web address line or click the 'clone and download' and copy that).
+Copy the web address for this repo 
+(from the web address line or click the 'clone and download' and copy that).
 
 ![](../fig/github_screenshot_upstream_repo.png)
 
 Then back in your terminal, navigate in the cloned repo and add the remote connection.
-For this command we must give the new connection a different nickname, where our original connection is 'origin'
+For this command we must give the new connection a different nickname, 
```

Comment:
Thanks!  Good idea.  I guess I must think of them as connections but the consistency is better.